### DISCUSSION TITLE
DependencyGraphDependency - GraphQL - Explorer

### DISCUSSION THREAD

**Discussion by jmreddy2106** [community/community#35795] [Apps, API and Webhooks]
I'm unable to see the option "dependencyGraphManifests" under the repository of GraphQl API explorer. However, from the Python API, I'm able to hit get the data.  Can anyone suggest how to get the dependency graph from the API in JavaScript? 
`{
  repository(name: "repo_name", owner: "owner_name") {
    description
    forkCount
    forks(first: 10) {
      totalCount
    }
    name
    releases(first: 10) {
      totalCount
      nodes {
        name
        url
      }
    }
    languages(first: 10) {
      totalCount
      nodes {
        name
        color
      }
    }
    licenseInfo {
      name
      url
      spdxId
    }
    nameWithOwner
    primaryLanguage {
      name
    }
    sshUrl
    watchers(first: 10) {
      edges {
        node {
          id
          name
        }
      }
      totalCount
    }
    assignableUsers(first: 10) {
      edges {
        node {
          id
          name
        }
      }
      totalCount
    }
  }
}`

**Comment by rlnchow** [comment]
@jmreddy2106 - 
`dependencyGraphManifests` is under Preview. 
Provide a custom media in the `Accept` header:  `application/vnd.github.hawkgirl-preview+json`
PS: [Reference](https://docs.github.com/en/graphql/overview/schema-previews#access-to-a-repositories-dependency-graph-preview)
